lib/api/test/unit/units/validateBody/isJson.test.js

```diff
@@ -0,0 +1,29 @@
+const { assert } = require('chai');
+const mediaTyper = require('media-typer');
+const { isJson } = require('../../../../units/validateBody');
+
+describe('isJson', () => {
```

Comment:
Airbnb says this should be isJSON - https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

Reply:
I don't see them covering acronyms being _within_ a variable. They do allow to use acronyms in lowercase, but how about cammelCase then?

```
isJson = cammelCase(is + json)
```

Reply:
I think what they're trying to say is that `isJSON` is better than `isJson` although the later is correct according the camelCase definition. As I understand it, what they are trying to say is that `isHTTPtransaction` is better than `isHttpTransaction` as well, for readability reasons.

Reply:
I'm not strongly opinionated about this and I'm fine if we're not strict about it.